core/sdk/src/main/java/com/mongodb/stitch/core/auth/internal/CoreStitchAuth.java

```diff
@@ -479,9 +531,9 @@ private StitchUserT processLoginResponse(
       // will keep the currently logged in user logged in if the profile request failed, and in
       // this particular edge case the user is linked, but they are logged in with their older
       // credentials.
```

Comment:
[task] update the comment for this block of code, since it's not just for link requests anymore